packages/headless/src/controllers/insight/query-summary/headless-insight-query-summary.ts

```diff
@@ -1,32 +1,17 @@
 import {InsightEngine} from '../../../app/insight-engine/insight-engine';
-import {Controller} from '../../controller/headless-controller';
 import {
   buildCoreQuerySummary,
-  CoreQuerySummaryState,
+  QuerySummary,
+  QuerySummaryState,
 } from '../../core/query-summary/headless-core-query-summary';
 
-/**
- * The `InsightQuerySummary` controller provides information about the current query and results (e.g., "Results
- * 1-10 of 123").
- * */
-export interface InsightQuerySummary extends Controller {
-  /** The state relevant to the `InsightQuerySummary` controller.*/
-  state: InsightQuerySummaryState;
-}
-
-/**
- * A scoped and simplified part of the headless state that is relevant to the `InsightQuerySummary` controller.
- */
-export interface InsightQuerySummaryState extends CoreQuerySummaryState {}
-
+export type {QuerySummary, QuerySummaryState};
 /**
  * Creates a `InsightQuerySummary` controller instance.
```

Comment:
Like with the others we should change the name in the jsdoc as well to be consistent.